app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,69 @@
+module Admin
+  class CategoriesController < AdminController
+    before_action :set_authorisation_status
```

Comment:
it could be part of AdminController, and duplication removed from rest of controllers